js/initcontext.go

```diff
@@ -45,8 +45,9 @@ type InitContext struct {
 	pwd         *url.URL
 
 	// Cache of loaded programs and files.
-	programs     map[string]programWithSource
-	exportsCache map[string]goja.Value
+	programs map[string]programWithSource
+	// mege this and the above
```

Comment:
```suggestion
	// merge this and the above
```
Can you document when it is expected to be done and what is the issue at the moment that blocks it, please?

Reply:
The blockign part is mostly that thsi will be quite a big refactor which in practice is done in https://github.com/grafana/k6/pull/2881